docs/authoring.md

```diff
@@ -65,23 +64,35 @@ Do this by adding the following to ``MyAuthoredComponent.csproj```
+**If you are going to write your own nuspec, i.e. not rely on automatic packaging** then the CsWinRT target that adds the hosting dlls to your package will not run, and you should make sure your nuspec contains the following ```file``` entries for ```MyAuthoredComponent``` (note: your TargetFramework may vary). If adding a file entry for ```Coords.targets``` does not work, then update the project file per the above example. 
 
 ``` nuspec
-  <files>
-    <file src="build\MyAuthoredComponent.CsWinRT.targets"             target="build\MyAuthoredComponent.CsWinRT.targets" />
-    <file src="build\MyAuthoredComponent.targets"                     target="build\MyAuthoredComponent.targets" />
-    <file src="buildTransitive\MyAuthoredComponent.CsWinRT.targets"   target="buildTransitive\MyAuthoredComponent.CsWinRT.targets" />
-    <file src="buildTransitive\MyAuthoredComponent.targets"           target="buildTransitive\MyAuthoredComponent.targets" />
-    <file src="lib\native\MyAuthoredComponent.dll"                    target="lib\native\MyAuthoredComponent.dll" />
-    <file src="lib\native\Microsoft.Windows.SDK.NET.dll"              target="lib\native\Microsoft.Windows.SDK.NET.dll" />
-    <file src="lib\native\WinRT.Host.Shim.dll"                        target="lib\native\WinRT.Host.Shim.dll" />
-    <file src="lib\native\WinRT.Runtime.dll"                          target="lib\native\WinRT.Runtime.dll" />
-    <file src="lib\net5.0-windows10.0.19041\MyAuthoredComponent.dll"  target="lib\net5.0-windows10.0.19041\MyAuthoredComponent.dll" />
-    <file src="metadata\MyAuthoredComponent.winmd"                    target="metadata\MyAuthoredComponent.winmd" />
-    <file src="runtimes\win-x64\native\WinRT.Host.dll"                target="runtimes\win-x64\native\WinRT.Host.dll" />
-    <file src="runtimes\win-x86\native\WinRT.Host.dll"                target="runtimes\win-x86\native\WinRT.Host.dll" />
-  </files>
+<files>
+  <file src="$(TargetDir)Coords.dll"                    target="lib\native\Coords.dll" />
+  <file src="$(TargetDir)Microsoft.Windows.SDK.NET.dll" target="lib\native\Microsoft.Windows.SDK.NET.dll" />
+  <file src="Generated Files\Coords.winmd" target="winmd\Coords.winmd" />
+   
+  <!-- Note: you must rename the CsWinRt.Authoring.Targets as follows -->
+  <file src="C:\Path\To\CsWinRT\NugetDir\buildTransitive\Microsoft.Windows.CsWinRT.Authoring.targets"   
+        target="buildTransitive\Coords.CsWinRT.targets" />
+   
+  <file src="C:\Path\To\CsWinRT\NugetDir\build\Microsoft.Windows.CsWinRT.Authoring.targets"       
+        target="build\Coords.CsWinRT.targets" />
+   
+  <!-- Include the managed DLLs -->
+  <file src="C:\Path\To\CsWinRT\NugetDir\lib\net5.0\WinRT.Host.Shim.dll"                                  

```

Comment:
Are these in TargetDir or no?

Reply:
No, because of this target: https://github.com/microsoft/CsWinRT/blob/30cdf4a0687d3ebbffe2dc5ab6770deaca9944a6/nuget/Microsoft.Windows.CsWinRT.targets#L33

We could add a condition to that target "CsWinRTComponent != true" then they would appear in the TargetDir... But I don't think we want to do that, since the managed DLL doesn't need these (thus shouldn't get them in the output dir)

Reply:
Makes sense.